### DISCUSSION TITLE
Participant AFQ versus GroupAFQ

### DISCUSSION THREAD

**Discussion by radhi-bhalerao** [yeatmanlab/pyAFQ#980] [General]
Hi, I'd like to understand the main differences between the ParticipantAFQ and GroupAFQ API objects. I was wondering because I ran ParticipantAFQ on some non-BIDS DTI data and received very similar FA profiles to my results from running GroupAFQ on that same data in BIDS format. I also wanted to confirm whether preprocessing via a pipeline like QSIprep is absolutely necessary if we intend to use AFQ to retrieve tract-specific FA data for a set of patients. Thanks!

**Comment by arokem** [comment]
Hi @radhi-bhalerao : thanks for asking this question! Since this is not an issue that requires a change to the code, I converted it to a discussion. Indeed, the main difference between ParticipantAFQ and GroupAFQ objects is the interaction with the BIDS standard. In fact, after figuring out where the data is based on the BIDS structure, GroupAFQ passes that information directly to a ParticipantAFQ object (that happens [here](https://github.com/yeatmanlab/pyAFQ/blob/master/AFQ/api/group.py#L294)) and subsequent processing is identical, so it would be surprising if your results were any different!

Regarding your second question: yes, we strongly recommend performing pre-processing of the data before running pyAFQ. Diffusion MRI data is sensitive to many types of artifacts, including from motion and from eddy motion, as well as noise and Gibbs ringing. It is very important to deal with these artifacts before running pyAFQ.
